Try

minghuaw opened the pull request.

simonwuelker commented:
You do not need to open a PR for try runs - just run `mach try wpt`. The resulting run is under the "actions" tab in your fork.

minghuaw committed:
remove success boolean from WebFontLoaded message

Signed-off-by: Minghua Wu <michael.wu1107@gmail.com>

minghuaw committed:
remove success boolean from WebFontLoaded message (linux-wpt)

{"fail_fast": false, "matrix": [{"name": "Linux (WPT)", "workflow": "linux", "wpt": true, "profile": "release", "unit_tests": false, "build_libservo": false, "bencher": false, "coverage": false, "build_args": "", "wpt_args": "", "number_of_wpt_chunks": 20}]}
